utility/src/main/java/org/oppia/android/util/parser/TestGlideImageLoader.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.util.parser
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import org.oppia.android.util.caching.AssetRepository
+import org.oppia.android.util.caching.CacheAssetsLocally
+import javax.inject.Inject
+
+/** A [TestGlideImageLoader] that is used in tests. */
```

Comment:
Please describe what this class does to make it clearer to readers when to use it.

Reply:
Done.